mailgun/views.py

```diff
@@ -10,3 +13,18 @@
 @require_http_methods(['GET'])
 def auth_error(request):
     return JsonResponse({'error': 'Failed to authenticate request.'}, status=401)
+
+
+@csrf_exempt
+@authenticate()
+@require_http_methods(['POST'])
+def log_post_data(request):
```

Comment:
I intentionally left it generic so it could be used to log  any post info as there isn't anything specific to webhooks here..